api/src/pcapi/routes/auth/blueprint.py

```diff
@@ -0,0 +1,24 @@
+from flask import Blueprint
+from flask_cors import CORS
+
+from pcapi import settings
+from pcapi.routes.auth import utils
+from pcapi.serialization.spec_tree import ExtendedSpecTree
+from pcapi.serialization.utils import before_handler
+
+
+auth_blueprint = Blueprint("auth", __name__, template_folder="templates")
+CORS(
+    auth_blueprint,
+    origins=settings.CORS_ALLOWED_ORIGINS_NATIVE,
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** Pas sur que ce soit le bon origin pour nous
Ils ne seront pas depuis l'app‏

Reply:
je change mais ça va dépandre de où on met nos route est ce qu' on laisse le lien du type: backend.passculture.app/auth/discord/signin
ou on veux quelque chose du genre auth.passculture.app
 pour le momment je considère que c'est dans le backend

Reply:
Yes je suis d'accord, je pense qu'on peut laisser backend.passculture.***